src/codebert/mod.rs

```diff
@@ -0,0 +1,72 @@
+//! # RoBERTa: A Robustly Optimized BERT Pretraining Approach (Liu et al.)
```

Comment:
I am unsure this module is required - tokenization and modeling seem to be identical for RoBERTa and CodeBERT